app/controllers/RepaymentTypeController.scala

```diff
@@ -65,21 +65,21 @@ class RepaymentTypeController @Inject() (
         value =>
           for {
             updatedAnswers: UserAnswers <- Future.fromTry(request.userAnswers.set(RepaymentTypePage, value))
-            updatesAnswersWithWhomToPay <- {
-              request.userAnswers.get(ClaimantTypePage) match {
+            updatedAnswers <- {
+              updatedAnswers.get(ClaimantTypePage) match {
                 case Some(Importer) => Future.fromTry(updatedAnswers.set(WhomToPayPage, models.WhomToPay.Importer))
                 case _              => Future.successful(updatedAnswers)
               }
             }
-            updatedAnswersWithCMA <- {
-              request.userAnswers.get(RepaymentTypePage) match {
-                case Some(CMA) => Future.fromTry(updatesAnswersWithWhomToPay.remove(BankDetailsPage))
-                case _         => Future.successful(updatesAnswersWithWhomToPay)
+            updatedAnswers <- {
+              updatedAnswers.get(RepaymentTypePage) match {
```

Comment:
This fixes a bug in the original implementation.   Bank details should not be present if the repayment method is CMA